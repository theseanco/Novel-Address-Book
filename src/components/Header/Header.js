import React from 'react';

const Header = () => {
  return (
    <header>
      <h1>Address Book App</h1>
    </header>
  )
}

export default Header;