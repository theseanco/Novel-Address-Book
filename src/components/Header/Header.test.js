import React from 'react';
import { render } from '@testing-library/react';
import Header from './Header';

describe('header tests', () => {

  it('should return the app title in a heading', () => {
    const { getByRole } = render(<Header />);
    expect(getByRole('heading')).toHaveTextContent('Address Book App')
  })

  // Snapshot test for the header component as it does not have stateful UI logic
  it('should match snapshot', () => {
    const { container } = render(<Header />);
    expect(container.firstChild).toMatchSnapshot();
  })
});