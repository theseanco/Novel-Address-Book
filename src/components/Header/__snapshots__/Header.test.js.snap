// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header tests should match snapshot 1`] = `
<header
  data-testid="header"
>
  <h1>
    Address Book App
  </h1>
</header>
`;
